docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -715,15 +750,21 @@ public class NativeBearerTokenAuthenticationIT extends BearerTokenAuthentication
 }
 ----
 
-Please see xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] for more information about the way it is initialized and configured.
+For more information about initializing and configuring Dev Services for Keycloak, see the xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] guide.
 
 [[integration-testing-keycloak]]
-==== KeycloakTestResourceLifecycleManager
+==== `KeycloakTestResourceLifecycleManager`
+
+You can also use `KeycloakTestResourceLifecycleManager` for integration testing with Keycloak.
 
-If you need to do some integration testing against Keycloak then you are encouraged to do it with <<integration-testing-keycloak-devservices,Dev Services For Keycloak>>.
-Use `KeycloakTestResourceLifecycleManager` for your tests only if there is a good reason not to use `Dev Services for Keycloak`.
+[IMPORTANT]
+====
+Where possible, use the method described in <<integration-testing-keycloak-devservices,Dev Services For Keycloak>> instead of using `KeycloakTestResourceLifecycleManager`, unless you have specific requirements for using this method.
+====
 
-Start with adding the following dependency:
```

Comment:
Please keep this line or as earlier start with `First, add the following dependency:`